MultiServer.py

```diff
@@ -261,6 +260,9 @@ def _load_game_data(self):
             self.non_hintable_names[world_name] = world.hint_blacklist
 
     def _init_game_data(self):
+        # WebHostContext
+        if type(self) is Context:
+            self._load_game_data()
         for game_name, game_package in self.gamespackage.items():
```

Comment:
This is confusing, and looks like a hack. I think it needs a lot more comment explaining what's going on, and why we don't want this load in any subclass.

Reply:
It honestly is a bit of a hack. Currently MultiServer loads the network data during startup but before it loads the multidata file. This is fine for customserver, since it uses static data where all worlds are loaded already, but if we want to lazy load in MultiServer then the loading has to be moved later. The specific problem is that customserver overrides `_load_game_data` to use its static data, and it deletes that data during startup.